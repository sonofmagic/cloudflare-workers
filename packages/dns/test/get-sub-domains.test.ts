import { getSubDomains } from '@/get-sub-domains'
import { isCI } from 'ci-info'

describe.skipIf(isCI)('get-sub-domains', () => {
  it('sonofmagic.top', async () => {
    const subDomains = await getSubDomains('sonofmagic.top')
    expect(subDomains).toMatchSnapshot()
  })
})
