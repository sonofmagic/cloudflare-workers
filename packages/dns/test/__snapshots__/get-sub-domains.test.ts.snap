// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`get-sub-domains > sonofmagic.top 1`] = `
[
  "docs.host.sonofmagic.top",
  "api.host.sonofmagic.top",
  "sonofmagic.top",
  "host.sonofmagic.top",
  "dev.host.sonofmagic.top",
]
`;
